python/dlr/api.py

```diff
@@ -7,13 +7,6 @@
 import logging
```

Comment:
Can we remove this import and replace all remaining logging() reference in the file with neo_logger?

Reply:
Done